### PR TITLE
osbuild release 21 and koji-osbuild 1

### PR DESCRIPTION
**Pull request by gicmo**
Add news entries for the osbuild 21 and kojib-osbuild 1 releases.

### CODE REVIEW

File: _collections/_news/2020-10-06-release-osbuild-21.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+caption: "Release of osbuild 21"
+author: "Christian Kellner"
+categories: [fedoraplanet]
+---
+We are happy to announce version 21 of *osbuild*. This release adds
+support for running on Fedora 34. Errors while setting up the
+container are now reported. A lot work has been done to make the
+code better to read.
```
Comment (dvdhrm) [inline _collections/_news/2020-10-06-release-osbuild-21.md]:
```suggestion
We are happy to announce version 21 of *osbuild*. This release adds
support for running on Fedora 34. Furthermore, internal errors during
container setup are now properly reported. And lastly, a lot of work has
been done to improve the code readability.
```

Comment (dvdhrm) [reply _collections/_news/2020-10-06-release-osbuild-21.md]: Just a suggestion^^

**Review by dvdhrm — APPROVED** [review]

File: _collections/_news/2020-10-06-release-koji-osbuild-1.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+caption: "First release of koji-osbuild"
+author: "Christian Kellner"
+categories: [fedoraplanet]
+---
+We are happy to announce the first release *koji-osbuild*, a new project
```
Comment (dvdhrm) [inline _collections/_news/2020-10-06-release-koji-osbuild-1.md]:
```suggestion
We are happy to announce the first release of *koji-osbuild*, a new project
```

File: _collections/_news/2020-10-06-release-koji-osbuild-1.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+to integrate *osbuild-composer* with [koji](https://pagure.io/koji), the
+build and tracking system primarily used by Fedora Project and Red Hat.
```
Comment (dvdhrm) [inline _collections/_news/2020-10-06-release-koji-osbuild-1.md]:
```suggestion
build and tracking system primarily used by the Fedora Project and Red Hat.
```

File: _collections/_news/2020-10-06-release-koji-osbuild-1.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  * Unit tests currently cover around 92% of the plugins.
+
+  * Detailed instructions on how to run a local test setup
+    are included in `HACKING.md`.
+
+
+Contributions from: Christian Kellner, Lars Karlitski &
+                    Tom Gundersen
+
+
```
Comment (dvdhrm) [inline _collections/_news/2020-10-06-release-koji-osbuild-1.md]:
```suggestion

```

This PEP-8 is spreading everywhere!

**Review by dvdhrm — APPROVED** [review]
Some more suggestions. Ignore them if you wish ;)

**Comment by gicmo** [comment]
>  Ignore them if you wish ;)

Never! All incorporated! Thanks a lot ;)

**Review by dvdhrm — APPROVED** [review]

**Pull request merged**